### PR TITLE
dockered owner

### PR DESCRIPTION
**Pull request by Qwtdgh**
process owner.sh in docker

### CODE REVIEW

**Comment by Qwtdgh** [comment]
@SongY123

File: adapter/adapter-csv/pom.xml [hunk 1]
```diff
@@ -71,6 +71,38 @@
...
+      </plugin>
+
+      <plugin>
+        <groupId>com.spotify</groupId>
+        <artifactId>docker-maven-plugin</artifactId>
+        <version>1.2.0</version>
+        <configuration>
+          <imageName>${project.artifactId}</imageName>
+          <dockerDirectory>adapter/adapter-csv/src/main/docker</dockerDirectory>
+          <dockerHost>http://127.0.0.1:2375</dockerHost>
```
Comment (SongY123) [inline adapter/adapter-csv/pom.xml:87]: Do not specify docker host here.

File: owner/pom.xml [hunk 2]
```diff
@@ -107,6 +112,47 @@
...
+      </plugin>
+
+      <plugin>
+        <groupId>com.spotify</groupId>
+        <artifactId>docker-maven-plugin</artifactId>
+        <version>1.2.0</version>
+        <configuration>
+          <imageName>${project.artifactId}</imageName>
+          <dockerDirectory>owner/src/main/docker</dockerDirectory>
+          <dockerHost>http://127.0.0.1:2375</dockerHost>
```
Comment (SongY123) [inline owner/pom.xml:128]: Do not specify docker host here.

File: owner/src/main/docker/Dockerfile
Comment (SongY123) [inline owner/src/main/docker/Dockerfile:1]: Add a new folder named with 'docker-build' in project root path and move this Dockerfile to the folder.

**Review by SongY123 — CHANGES_REQUESTED** [review]

File: adapter/adapter-csv/pom.xml (hunk 1 above)
Comment (Qwtdgh) [reply adapter/adapter-csv/pom.xml]: resolve

File: owner/src/main/docker/Dockerfile
Comment (Qwtdgh) [reply owner/src/main/docker/Dockerfile:1]: resolve

**Review by SongY123 — APPROVED** [review]

**Pull request merged**